docs/source/reference/spawners.md

```diff
@@ -25,6 +25,8 @@ Some examples include:
   run without being root, by spawning an intermediate process via `sudo`
 - [BatchSpawner](https://github.com/jupyterhub/batchspawner) for spawning remote
   servers using batch systems
+- [YarnSpawner](https://github.com/jcrist/yarnspawner) for spawning notebook
```

Comment:
If you would like to maintain this on the jupyterhub org, I'd be happy to move it over and preserve your ownership. No worries if not.